gradle.deploy.properties

```diff
@@ -1,49 +1,37 @@
 runName=beamville
 beamBranch=develop
 # beamCommit=dc4b0b05a3885357c7b5b65fb91181c8163aa9b6
-beamConfigs=test/input/beamville/beam.conf
-instanceType=r5.2xlarge
 dataBranch=develop
 # dataCommit=HEAD
 
+# comma-separated list of configs (or single config) which will be run on an AWS instances (one instance per config)
+beamConfigs=test/input/beamville/beam.conf
+
+instanceType=r5.2xlarge
+
 # shutdownBehaviour = stop | terminate
 shutdownBehaviour=terminate
+
 s3Backup=true
 
-runHelics=false
-profiler_type=cpumem
+# path to a shell script to start cosimulation, expected to be related to src/main/bash/
+# for example: helics/run_pydss_federate.sh or helics/run_site_power_controller.sh
+cosimulationShellScript=""
 
 # Run Jupyter Notebook together with BEAM
-runJupyter = false
+runJupyter=false
 
-# budgetOverride=false
-# forcedMaxRAM=700
-
-# storageSize (in GiB) =  any number between 64 and 256
-storageSize=128
+# type of profiler - empty or cpu or cpumem
 profiler_type=cpumem
 
-#r5.8xlarge (32/256)
-#c5.9xlarge (36/72)    -> 5 instances -> $1.53 per Hour
-#m4.10xlarge (40/160)   -> 5 -> $2.00 per Hour
-#m5.12xlarge (48/192)   -> 3 instances -> $2.304 per Hour
-#m5d.12xlarge (48/192)  -> 5 -> $2.712 per Hour
-
-#m4.16xlarge (64/256)   -> 20 -> $3.20 per Hour
-#r5.12xlarge (48/384)   -> 3 -> $3.024 per Hour
-#c5.18xlarge (72/144)   -> 5 -> $3.06 per Hour
-#c5d.18xlarge (72/144)  -> 5 -> $3.456 per Hour
-#r5d.12xlarge (48/384)  -> 3 -> $3.456 per Hour
-#h1.16xlarge (64/256)   -> 1 -> $3.744 per Hour
-
-#m5.24xlarge (96/384)   -> 2 -> $4.608 per Hour
-#i3.16xlarge (64/488)   -> 1 -> $4.992 per Hour
-#i3.metal (72/512)      -> 5 -> $4.992 per Hour
+# budgetOverride=false
```

Comment:
Removing prices is intentional?

Reply:
yes, the prices might change in the near future and we usually use very few types of instances anyway